src/clj/game/cards-agendas.clj

```diff
@@ -33,6 +33,27 @@
                                                                              (resolve-ability state side (abt 1 n) card nil))}
                                                                card nil)))}}})
 
+   "Advanced Concept Hopper"
+   {:events
+    {:run
+     {:req (req (first-event state side :run))
+      :effect (effect (show-wait-prompt :runner "Corp to use Advanced Concept Hopper")
+                      (resolve-ability
+                        {:optional
+                         {:player :corp
+                          :prompt "Use Advanced Concept Hopper to draw 1 card or gain 1 [Credits]?" :once :per-turn
+                          :yes-ability {:prompt "Draw 1 card or gain 1 [Credits]?"
+                                        :choices ["Draw 1 card" "Gain 1 [Credits]"]
+                                        :effect (req (if (= target "Gain 1 [Credits]")
+                                                       (do (gain state :corp :credit 1)
+                                                           (system-msg state :corp
+                                                                       (str "uses Advanced Concept Hopper to gain 1 [Credits]")))
+                                                       (do (draw state :corp)
```

Comment:
Maybe a prompt for allowing Corp to trigger any ACH on click, with a "Done"  button to cancel action if they want to use fewer than they have scored. Like Team Sponsorship, except with a prompt?